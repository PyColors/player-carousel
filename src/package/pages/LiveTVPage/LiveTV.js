import React from "react";

const LiveTV = () => (
    <h3>
        Live TV Page
    </h3>
);
export default LiveTV;
