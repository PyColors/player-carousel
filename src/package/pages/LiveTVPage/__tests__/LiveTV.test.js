import React from "react";
import LiveTV from "../LiveTV";

it("Should renders LiveTV page without exploding", () => {
    const wrapper = shallow(<LiveTV>Hello Jest!</LiveTV>);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper).toHaveLength(1);
});
