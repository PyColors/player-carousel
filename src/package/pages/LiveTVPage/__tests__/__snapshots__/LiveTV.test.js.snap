// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should renders LiveTV page without exploding 1`] = `
<h3>
  Live TV Page
</h3>
`;
